containers/EmpSponsors.ts

```diff
@@ -18,56 +22,56 @@ interface EmpMap {
   [address: string]: SponsorMap;
 }
 
+interface FinancialContractQuery {
+  id: string;
+  sponsorPositions: PositionState;
+}
+
 const useEmpSponsors = () => {
   const { block$ } = Connection.useContainer();
   const { contract: emp } = EmpContract.useContainer();
+  const { loading, error, data } = useQuery(ACTIVE_POSITIONS);
```

Comment:
What happens when the state changes? Does this just pull from the cache after the first query and never update again? Or is there a long-running poll that updates `data` every so often?

If data does get updated, does loading ever become `true` again after the first load?

Reply:
> If data does get updated, does loading ever become true again after the first load?

No its only `true` on the first query (i.e. when the cache is empty I'd assume)

> What happens when the state changes? Does this just pull from the cache after the first query and never update again? Or is there a long-running poll that updates data every so often?

I don't think the query will fire again once loaded. But, I think I can set a `poll: ms` option that makes the query refresh periodically. We can also import a `networkStatus` from `useQuery` that will live alert us when polling is done by setting a `notifyOnNetworkStatusChange: true` option. 

I have the `queryPositions()` run now whenever `emp` or `networkStatus` changes. I've set `poll:1000ms`. 

More details [here](https://www.apollographql.com/docs/react/data/queries/#polling)

Reply:
Actually: I've decided to set `poll: 5000` since position state shouldn't change at a high frequency.

I also removed the `networkStatus` and the `notifyOnNetWorkStatusChange` vars and instead call `queryPositions()` whenever `emp` or `data` changes.